src/Components/Scenes/SearchResults/CheckBoxButton/CheckBoxButton.js

```diff
@@ -21,19 +22,21 @@ const CheckBoxButton = props => {
 
   return (
     <React.Fragment>
```

Comment:
fyi, you can now use a short hand for fragments
`<></>` instead of `<React.Fragment>`
ref: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
I know that but I never in favor of that. But Robert is using that `<></>` 
I am using `React.Fragment` . We need to be consist. 
I will be start using this `<></>`

Reply:
There may be times when we need to provide a key, and may need to use React.Fragment.   But if it's just an empty fragment, my preference is the short hand.  We could always take it to a team vote.